src/app/2025/(home)/_components/Stats/AnimatedCounter.tsx

```diff
@@ -0,0 +1,35 @@
+"user-client";
```

Comment:
```suggestion
"use client";
```
[ref](https://react.dev/reference/rsc/use-client)